lib/models/addon.js

```diff
@@ -375,7 +375,9 @@ let addonProto = {
 
           throw new SilentError(
             'Your names in package.json and index.js should match. ' +
-            `The addon in ${pathToDisplay} currently have '${parentName}' in package.json and '${this.name}' in index.js.`);
+            `The addon in ${pathToDisplay} currently have '${parentName}' in package.json and '${this.name}' in index.js.` +
+            'For more information about this workaround, see: https://github.com/ember-cli/ember-cli/pull/7950. ' +
+            'Until ember-cli v3.9, this error can be disabled by setting env variable EMBER_CLI_IGNORE_ADDON_NAME_MISMATCH to "true"');
```

Comment:
@mike-north can you swap the last two lines?

Reply:
done